src/styles/media.scss

```diff
@@ -0,0 +1,11 @@
+@media (min-width: 600px) {
+  .container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (min-width: 900px) {
+  .container {
+    grid-template-columns: repeat(3, minmax(300px, 1fr));
+  }
+}
```

Comment:
Media queries should be placed with their selector in the same file

Reply:
delete this file, it's redundant